packages/config/config/devices/0x0438/k8.json

```diff
@@ -0,0 +1,61 @@
+// Namron K8
+// 4-Channel Wireless Switch
+{
+	"manufacturer": "Namron",
+	"manufacturerId": "0x0438",
+	"label": "K8",
+	"description": "4-Channel Wireless Switch",
+	"devices": [
+		{
+			"productType": "0x0300",
+			"productId": "0xa305",
+			"zwaveAllianceId": 3693
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Button 1",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Button 2",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Button 3",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Button 4",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Factory Reset",
+			"valueSize": 2,
+			"minValue": 21930,
+			"maxValue": 21930,
+			"defaultValue": 0,
```

Comment:
That doesn't make sense. And we usually use options for a factory reset (also applies to the other files).

Reply:
The manual says "set to 21930 to factory reset". What options do you usually use for factory reset? If you could link an example I will fix it for these files 🙂

Reply:
I meant the default value needs to be in the range defined by min/max - we should probably add an automated check for that. Let me see if I find one...

Reply:
Here's one:
https://github.com/zwave-js/node-zwave-js/blob/32b5f9fc76afc21439d5f88b534cb3654156612b/packages/config/config/devices/0x0090/templates/kwikset_template.json#L258-L279